.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## FreeCodeCamp - Wiki Pull Request template
+
+Make sure your contribution addresses the points mentioned below:
+
+Filenames:
+- Your filename shoud NOT contain any special chars or spaces.
+- Use only "-" to separate words in the filename.
+- Filenames should have extention ".md"
+
+Formatting:
+- Ensure that you use markdown syntax for articles.
+- Checkout the guides and cheetsheets for reference:
+1. https://guides.github.com/features/mastering-markdown/
+2. https://help.github.com/articles/working-with-advanced-formatting/
+
+Commits and Squashing:
+- Your PR/Submission should have only one commit.
+
+- For multiple commits follow the below guide to squash.
+  https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one
```

Comment:
Make the link like the others, displaying the full url is no needed. Also Pull Request instead of PR. Later they will familiarize with our terms.